simple_message/src/messages/robot_status_message.cpp

```diff
@@ -66,6 +66,7 @@ bool RobotStatusMessage::init(industrial::simple_message::SimpleMessage & msg)
   bool rtn = false;
   ByteArray data = msg.getData();
   this->init();
+  this->setCommType(msg.getCommType());

```

Comment:
Sorry but I don't understand what you mean. Maybe I'm wrong, but how can I retrieve the comm_type_ value (private defined) without using those methods?